src/client/pages/VergunningDetail/EigenParkeerplaats.tsx

```diff
@@ -0,0 +1,81 @@
+import type { EigenParkeerplaats as EigenParkeerplaatsType } from '../../../server/services';
+import { defaultDateFormat } from '../../../universal/helpers';
+import InfoDetail from '../../components/InfoDetail/InfoDetail';
+import { Location } from './Location';
+
+function getVerzoekType(vergunning: EigenParkeerplaatsType) {
+  switch (true) {
+    case vergunning.isNewRequest:
+      return 'Nieuwe aanvraag';
+    case vergunning.isCarsharingpermit:
+      return 'Autodeelbedrijf';
+    case vergunning.isLicensePlateChange:
+      return 'Kentekenwijziging';
+    case vergunning.isRelocation:
+      return 'Verhuizing';
+    case vergunning.isExtension:
+      return 'Verlenging';
+  }
+}
+
+export function EigenParkeerplaats({
+  vergunning,
+}: {
+  vergunning: EigenParkeerplaatsType;
+}) {
+  const isVerleend = vergunning.processed && vergunning.decision === 'Verleend';
+  const isAfgehandeld = vergunning.processed;
+
+  return (
+    <>
+      <InfoDetail label="Kenmerk" value={vergunning.identifier || '-'} />
+      <InfoDetail label="Verzoek" value={getVerzoekType(vergunning)} />
+
+      <Location
+        label="Adres"
+        location={`${vergunning.streetLocation1} ${vergunning.housenumberLocation1}`}
+      />
+      <InfoDetail label="Soortplek" value={vergunning.locationkindLocation1} />
```

Comment:
`locationkindLocation1` is een vrij nietszeggende naam. Kun je niet beter iets doen als: `typeOfLocation1` en `typeOfLocation2` ?

Reply:
Of de data anders binnen laten komen? 

```
locations: [{ type: 'xx', address: 'Straat 1'},{ type: 'yy', address: 'Straat 2'}]
```

Dat maakt een en ander wat leesbaarder.

Reply:
`const [location1, location2] = source.locations`